driver-core/src/test/functional/com/mongodb/internal/operation/CommandBatchCursorFunctionalTest.java

```diff
@@ -101,6 +102,55 @@ void cleanup() {
         });
     }
 
+    @Test
+    @DisplayName("should exhaust cursor with multiple batches")
+    void shouldExhaustCursorAsyncWithMultipleBatches() {
```

Comment:
This method should be renamed, as this is not an async test. The same applies to `shouldExhaustCursorAsyncWithClosedCursor`, `shouldExhaustCursorAsyncWithEmptyCursor`.